src/openqdc/datasets/waterclusters3_30.py

```diff
@@ -51,13 +53,13 @@ class WaterClusters(BaseDataset):
 
     # Energy in hartree, all zeros by default
     atomic_energies = np.zeros((MAX_ATOMIC_NUMBER,), dtype=np.float32)
-
+    # need to know where to find the data
```

Comment:
https://sites.uw.edu/wdbase/database-of-water-clusters/

Reply:
Ok so you downloaded the entire dataset made from the flexible potential. WIth the isolated atom energy PR I'll have a manual check of the data as with the other unsure dataset